src/main/java/hudson/plugins/git/browser/AssemblaWeb.java

```diff
@@ -14,6 +14,7 @@
 import org.kohsuke.stapler.QueryParameter;
 import org.kohsuke.stapler.StaplerRequest;
 
+import javax.annotation.Nonnull;
```

Comment:
Can we stick to the findbugs ones?

Reply:
I used whatever annotation that the methods extended does in an effort to try and not make too much of a mess :)